both/methods.js

```diff
@@ -153,16 +153,25 @@ Meteor.methods({
 
     Meteor.call('refreshRoundCodes', competitionId, eventCode);
   },
-  addNonEventRound: function(competitionId, round) {
+  addScheduleEvent: function(competitionId, eventData, roundId) {
     check(competitionId, String);
     throwIfCannotManageCompetition(this.userId, competitionId);
-    return Rounds.insert({
+
+    var round = Rounds.findOne(roundId);
+
+    return ScheduleEvents.insert({
       competitionId: competitionId,
-      title: round.title,
-      startMinutes: round.startMinutes,
-      durationMinutes: round.durationMinutes,
+      roundId: (round ? round._id : null),
```

Comment:
I investigated this a bit.

`console.log()` in Chrome does give a link to the source line where it's thrown. I think that's all you need. As you say, the stack trace past that is just internals.

Logging the error gives you the error code, message and details, but not the server side stack trace. You can probably find those in the server log I assume exists, but in practice it should be clear what went wrong.

Also, if you just put a `throw new Meteor.Error(313, "I DIED!");` in the Method, you'll get a confusing **Exception while simulating the effect of invoking 'addRound'** exception printout.